src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/import.py

```diff
@@ -76,7 +79,8 @@ def acr_import(cmd,
         except NoTTYException:
             raise CLIError(NO_TTY_ERROR)
 
-    image_source = ImportSource(resource_id=source_registry, source_image=source_image)
+    source_credentials = ImportSourceCredentials(username=source_registry_username, password=source_registry_password)
```

Comment:
Do you need to create this `ImportSourceCredentials` if `username` and `password` are empty?

Do you need to check if one of `username` or `password` is empty?